packages/fixie/src/agent.ts

```diff
@@ -260,6 +262,39 @@ export class FixieAgent {
     return config as AgentConfig;
   }
 
+  private static inferRuntimeParametersSchema(agentPath: string): TJS.Definition | null {
+    // If there's a tsconfig.json file, try to use Typescript to produce a JSON schema
+    // with the runtime parameters for the agent.
+    const tsconfigPath = path.resolve(path.join(agentPath, 'tsconfig.json'));
+    if (!fs.existsSync(tsconfigPath)) {
+      term.yellow('⚠️ tsconfig.json not found. Your agent will not support runtime parameters.\n');
```

Comment:
When giving a "not found" message, it's always nice to include the full path that was checked.